extensions-core/google-extensions/src/test/java/org/apache/druid/data/input/google/GoogleCloudStorageInputSourceTest.java

```diff
@@ -66,8 +67,9 @@
 
 public class GoogleCloudStorageInputSourceTest extends InitializedNullHandlingTest
 {
-  private static final long EXPECTED_MAX_LISTING_LENGTH = 1024L;
+  private static final int EXPECTED_MAX_LISTING_LENGTH = 10;
```

Comment:
nit: MAX_LISTING_LENGTH since we're mocking the maxListingLength() to this value

Reply:
fixed